core/domain/user_services.py

```diff
@@ -340,12 +356,14 @@ def get_users_settings(user_ids):
             result.append(UserSettings(
                 feconf.SYSTEM_COMMITTER_ID,
                 email=feconf.SYSTEM_EMAIL_ADDRESS,
+                role=feconf.ROLE_SUPER_ADMIN,
```

Comment:
This is not a super admin role. It is actually a sort of oppia-bot that fixes stuff up. I think just admin or moderator is sufficient.